packages/form/components/Field.astro

```diff
@@ -29,6 +30,9 @@ const hasErrors: boolean | null = !!control.errors?.length;
 			) :
 			control.type === 'dropdown' ? (
 				<DropdownControl control={control as Dropdown} />
+			) :
+			control.type === 'textarea' ? (
+				<TextAreaControl control={control} />
```

Comment:
This will then need `control as TextArea`, which is not the best, but we can improve on this later... I'm working on something like a control component resolver where we should not have to typecast with 'as'